documentation/jetty-10/index.html

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <html lang="en">
-<head><link rel="canonical" href="https://eclipse.dev/jetty/documentation/jetty-12/index.html"/>
```

Comment:
@jmcc0nn3ll I think I need to retain this `<link rel..` element, right?

Don't we have a tool to do that already?

Reply:
Figured it out.